s3/sc-portfolio-s3-basic-external.yaml

```diff
@@ -0,0 +1,34 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
We were not planning to offer an S3 portfolio to external users

Reply:
Ok. will remove external S3 portfolio